t/10-tests_overview.t

```diff
@@ -233,8 +233,8 @@ $t->get_ok('/tests/overview?distri=opensuse&version=13.1&groupid=1001&groupid=10
 $summary = get_summary;
 like(
     $summary,
-    qr/Summary of opensuse, opensuse test/i,
-    'multiple groups with no build specified yield latest build of first group'
+    qr/Summary of opensuse, opensuse test build 0091 /i,
```

Comment:
This extra space is to ensure we don't have more than one build in the string

Reply:
Why not use `$` there?

Reply:
Because then the regex shouldn't match

Reply:
```suggestion
    qr/Summary of opensuse, opensuse test build 0091[^,]/i,
```

Reply:
yes, that works. I have created https://github.com/os-autoinst/openQA/pull/3973 incorporating the latest review suggestions from here so please approve there as long as ilausuch did not update their PR